docs/modules/ROOT/pages/quarkus-azure-key-vault.adoc

```diff
@@ -5,17 +5,20 @@ include::./includes/attributes.adoc[]
 include::./includes/support.adoc[]
 
 https://azure.microsoft.com/en-us/products/key-vault[Azure Key Vault] is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, certificates, or cryptographic keys. 
-This extension allows you to create and retrieve secret from Azure Key Vault by injecting the following object inside your Quarkus application.
+This extension allows you to:
 
-* `com.azure.security.keyvault.secrets.SecretClient` 
-* `com.azure.security.keyvault.secrets.SecretAsyncClient` 
+* Create and retrieve secrets from Azure Key Vault by injecting the one or both of the following objects in a Quarkus application.
+    - `com.azure.security.keyvault.secrets.SecretClient` 
+    - `com.azure.security.keyvault.secrets.SecretAsyncClient` 
+    
+* Use secrets from the key vault directly inside your `application.properties`.
 
-The extension produces SecretClient and  SecretAsyncClient using https://learn.microsoft.com/java/api/overview/azure/identity-readme#defaultazurecredential[DefaultAzureCredential].
-Developers who want more control or whose scenario isn't served by the default settings should build client using other credential types.
+The extension produces `SecretClient` and `SecretAsyncClient` using https://learn.microsoft.com/java/api/overview/azure/identity-readme#defaultazurecredential[DefaultAzureCredential].
+Developers who want more control or whose scenario isn't served by the default settings should build their clients using other credential types.
 
 == Installation
 
-If you want to use this extension, you need to add the `io.quarkiverse.azureservices:quarkus-azure-services` extension first to your build file.
+Add the `io.quarkiverse.azureservices:quarkus-azure-services` extension first to your build file.
```

Comment:
should be `io.quarkiverse.azureservices:quarkus-azure-keyvault`?

Reply:
See fix in all doc https://github.com/quarkiverse/quarkus-azure-services/pull/266/commits/367d4de0bea48779cce9a57ff9dab55fcb74c99d